app/templates/content/cookies.html

```diff
@@ -73,6 +73,12 @@ <h3>Universal Analytics</h3>
                 <td>Helps us count how many people visit the Digital Marketplace by tracking if you’ve visited the service before
                 </td>
                 <td>2 years</td>
+            </tr>
+            <tr>
+                <td>_gid</td>
+                <td>Helps us count how many people visit the Digital Marketplace by tracking if you’ve visited the service before
```

Comment:
Should we maybe let users know more about how `_ga`cookie is different from `_gid`? How are they different generally? I mean, purpose stated for each is the same, but expiry time is different, why is that?

Reply:
Google doesn't distinguish between them so we have no way of knowing how they're different (see the link in description). Looks like the accepted solution has been to keep the entries identical.
https://github.com/alphagov/govuk-design-system/blob/master/src/cookies.md.njk
https://github.com/alphagov/paas-product-page/blob/master/views/cookies.erb
https://www.gov.uk/help/cookies